routes/index.js

```diff
@@ -0,0 +1,14 @@
+var express = require("express");
+var router = express.Router();
+
+
+//// Routes
+
+
+//// Landing Route
+router.get("/", function (req, res) {
+	res.render("landing");
+});
```

Comment:
_ES6 lecture_
Let's start using arrow functions for these kinds of things.
So this can be rewritten as:
```js
router.get("/", (req, res) => {
  res.render("landing");
});
```
or using implicit return
```js
router.get('/', (req, res) => res.render('landing'));
```

But again, we are not creating any views so remove this 🤕